labrad/node/__init__.py

```diff
@@ -36,6 +36,8 @@
                     along with all their subdirs.
   extensions (*s): what files to look at, e.g. ['.ini', '.py', '.exe']
   autostart (*s): list of servers to be autostarted
+  javapath (s): Path for java VM, e.g. "C:\\Java\\jre1.7.0\\bin\\java.exe".
+                Needed if java or scala servers are used.
```

Comment:
This is no longer used since we provide start scripts for the java/scala servers that invoke the jvm. The javapath key may still exist in the registry for old nodes, but there's no reason to keep it, and no need to document it.